packages/forklift-console-plugin/webpack.config.ts

```diff
@@ -150,6 +150,7 @@ const config: WebpackConfiguration & {
   },
   devServer: {
     static: ['./dist'],
+    watchFiles: ['./dist', '../packages/common/dist'],
```

Comment:
AFAIU this will not work until we start watching and recompiling the common package,  but we can do it in a followup

Reply:
suggestion: since this will not work until we add watch to the common package, let's drop this line, it will not work and make webpack watch files it should not ( it already watch this package files so ./dist is wrong, and common will not change because we don't have a watch on its src )

Reply:
@rszwajko can you drop this line and see if it still works ?
this lines should not change anything